spec/fixtures/npm-license-expression/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "fake-package"
+  "description": "Fake package available under various licenses",
+  "license": "(MIT OR Apache-2.0 OR AGPL-3.0+)",
```

Comment:
It will take some reworking of the logic in the Project class, but in a subsequent iteration, it'd be cool to be able to parse the expression and return multiple licenses, where we can match the SPDX ID.